packages/table/preview/index.html

```diff
@@ -52,6 +76,7 @@
     .sidebar {
       background-color: #EBF1F5;
       flex-basis: 300px;
+      flex-shrink: 0;
```

Comment:
i had some trouble getting the preview layout to work with the slowlayoutstack. i'd like to revisit these styles and solidify how we're going to recommend creating layout boundaries.

it might even be worth adding a `<LayoutBoundary>` react component that uses style attributes (to avoid any class overrides from breaking them)